### PR TITLE
Enhanced to get vxlan and openvswitch details

### PR DESCRIPTION
**Pull request by vishwanathjadhav**
Enhanced to get all the details of vxlan and openvswitch from `ip -s link`.

fix: https://github.com/RedHatInsights/insights-core/issues/2329

### CODE REVIEW

File: insights/parsers/ip.py [hunk 1]
```diff
@@ -144,6 +148,16 @@ def parse_tx_stats(line, d):
     d["tx_collsns"] = int(split_content[5])
 
 
+def parse_vxlan_details(line, d):
```
Comment (xiangce) [inline insights/parsers/ip.py]: The computing of the function is quite simple, better remove the function and move the line to the calling place.

File: insights/parsers/ip.py [hunk 2]
```diff
@@ -144,6 +148,16 @@ def parse_tx_stats(line, d):
...
+    split_content = line.split()
+    d['vxlan'] = split_content
+
+
+def parse_ovs_details(line, d):
```
Comment (xiangce) [inline insights/parsers/ip.py]: The same comment as above

File: insights/parsers/ip.py (hunk 1 above)
Comment (vishwanathjadhav) [reply insights/parsers/ip.py]: Done

File: insights/parsers/ip.py (hunk 2 above)
Comment (vishwanathjadhav) [reply insights/parsers/ip.py]: Done

**Pull request merged**